opencsp/app/fixed_pattern_deflectometry/lib/FixedPatternSetupCalibrate.py

```diff
@@ -217,6 +258,28 @@ def _calculate_camera_poses(self) -> None:
             self._rots_cams.append(Rotation.from_rotvec(rvec.squeeze()))
             self._vecs_cams.append(Vxyz(tvec))
 
+    def _print_camera_pose_reprojection_errors(self) -> None:
+        """Prints reprojection errors after finding camera
+        """
+        for idx_cam in range(len(self._images)):
+            # Collect inputs
+            rvecs = self._rots_cams[idx_cam].as_rotvec()[None, :]
+            tvecs = self._vecs_cams[idx_cam].data.T
+            pts_obj = self._pts_xyz_corners.data.T
+            camera_indices = np.zeros(
+                self._marker_corner_ids[idx_cam].size, dtype=int)
+            point_indices = self._marker_corner_idxs[idx_cam]
+            points2d = self._marker_corners_xy[idx_cam].data.T
+
+            # Calculate errors
+            errors = ph.reprojection_errors(
+                rvecs, tvecs, pts_obj, self._camera, camera_indices, point_indices, points2d)
+            errors = Vxy(errors.T)
+
+            # Print errors
+            print(
```

Comment:
Consider using `lt.info()` instead of `print()`:

```python
import opencsp.common.lib.tool.log_tools as lt

def say_hello():
    lt.info("Hello, world!")

def whisper_goodbye():
    lt.debug("Goodbye, world!")

if __name__ == "__main__":
    lt.logger(level=lt.log.INFO)
    say_hello()
    whisper_goodbye()
```

The two main advantages of this are:
- it handles levels for you
- logs are managed correctly when running on HPC clusters

Reply:
I really like this, I will definitely try to implement lt into these operations.

Reply:
Awesome! Code reviews ftw!